plugin.cpp

```diff
@@ -27,72 +28,77 @@ static const char * def_cfg = QUOTE({
 	"email_to" : {
 		"description" : "The address to send the alert to",
 		"type" : "string",
-		"default" : "alert.subscriber@dianomic.com",
+		"default" : "Notification alert subscriber <alert.subscriber@dianomic.com>",
```

Comment:
Why have you changed this default? Getting rid of the the name version and sign it like this will break backward compatibility.

Reply:
Actually that was my request. We understood now, that angle brackets are a must around email address when using curl smtp email. In general, just embedding display name in address should not break things.

Reply:
Bit if somebody is already using the 'To name" item that will now stop working when they upgrade

Reply:
@MarkRiddoch It's been reverted now